types/artifacts.d.ts

```diff
@@ -664,6 +667,8 @@ declare global {
       export interface TimingSummary {
         firstContentfulPaint: number;
         firstContentfulPaintTs: number | undefined;
+        firstContentfulPaintAllFrames: number;
```

Comment:
this one is the one I meant
```suggestion
        firstContentfulPaintAllFrames: number | undefined;
```

sorry :)

Reply:
Already done :)